source/fuzz/fuzzer_util.h

```diff
@@ -349,19 +349,20 @@ uint32_t MaybeGetStructType(opt::IRContext* ir_context,
                             const std::vector<uint32_t>& component_type_ids);
 
 // Recursive definition is the following:
-// - if |scalar_or_composite_type_id| is a result id of a scalar type - returns
-//   a result id of the following constants (depending on the type): int -> 0,
-//   float -> 0.0, bool -> false.
+// - if |scalar_or_composite_type_id| is a result id of a scalar type -
+// returns
+//   a result id of the following constants (depending on the type): int ->
+//   0, float -> 0.0, bool -> false.
 // - otherwise, returns a result id of an OpConstantComposite instruction.
 //   Every component of the composite constant is looked up by calling this
 //   function with the type id of that component.
 // Returns 0 if no such instruction is present in the module.
-// The returned id either participates in IdIsIrrelevant fact or not, depending
-// on the |is_irrelevant| parameter.
+// The returned id either participates in IdIsIrrelevant fact or not,
+// depending on the |is_irrelevant| parameter.
```

Comment:
Did you intend to reformat this?  It doesn't look like anything has changed content-wise, so better not to reformat it if not.  Can you revert (or clarify if I've misunderstood).

Reply:
Reverted